lib/hiera/config.rb

```diff
@@ -10,8 +10,8 @@ class << self
     # @return [Hash] representing the configuration.  e.g.
     #   {:backends => "yaml", :hierarchy => "common"}
     def load(source)
-      @config = {:backends => "yaml",
-                 :hierarchy => "common",
+      @config = {:backends => ["yaml"],
+                 :hierarchy => ["nodes/%{::trusted.certname}", "common"],
```

Comment:
Looks like it does https://travis-ci.org/puppetlabs/hiera/jobs/67921209#L428-L429